mid updating

jovo opened the pull request:
no need to review yet, this is still a draft

jovo committed:
mid updating

jovo committed:
Update agreements.md

jovo committed:
Update agreements.md

bdpedigo reviewed (commented):
suggestions and questions (if agreed upon i can implement)
- I feel discomfort at using "NeuroData family" rather than, say, "NeuroData team". Reason being that while I want to have a collegial/friendly/supportive environment (which I think we have) I also view this as employment, and would like to respect that boundary and have the language reflect that.
- I don't feel a full body yes to having the "Play" section as part of the agreements. "I commit to play by honoring rest, renewal and rhythm." While I love play, this is not actionable to me. 
- What is the difference between a commitment and an agreement? If the difference is minor, I think the organization would make more sense without that distinction. if the difference is not minor, we should say what it is? or how it is operationally different for us?

jovo committed:
updated intro

bdpedigo commented on content/about/agreements.md:
```diff
@@ -7,138 +7,222 @@ $order: 5
-Everyone is welcome to use either AWS or Azure as desired. Currently, we have free Azure credits, so that is the preference. For AWS, if you want to use it, the prerequisite is writing a [AWS Cloud Credits for Research grant](https://aws.amazon.com/research-credits/). Request just under $10,000 for small projects, and request up to $100,000 for larger ones. For the Brainlit project, we have an NI ...
+#### Genius
 
+> I commit to focusing the majority of my research [energy](http://a.co/aPrBEkq) on a single project in my  [zone of genius](https://www.amazon.com/dp/B07S7L6FV2?ref=KC_GS_GB_US), and to support and inspire others to fully express their creativity and live in their zone of genius.
```
> on a single project

In the past we had said 2 projects at a time - is this an intentional change? I am pro 1-2 projects like we have had in the past

Laknath1996 replied:
I'm also pro 1-2 projects

jovo replied:
I changed it because I am trying to be *less* controlling.  I guess I meant to write something more like:

> I commit to focusing the majority of my research [energy](http://a.co/aPrBEkq) on research projects in my  [zone of genius](https://www.amazon.com/dp/B07S7L6FV2?ref=KC_GS_GB_US).

But maybe you are right, better to say 'a few' rather than 'a single'?

jovo commented:
> * I feel discomfort at using "NeuroData family" rather than, say, "NeuroData team". Reason being that while I want to have a collegial/friendly/supportive environment (which I think we have) I also view this as employment, and would like to respect that boundary and have the language reflect that.

ok, i don't feel strongly either way.

> * I don't feel a full body yes to having the "Play" section as part of the agreements. "I commit to play by honoring rest, renewal and rhythm." While I love play, this is not actionable to me.

the actions associated with this are:
- recess
- vacations
- etc. 

> * What is the difference between a commitment and an agreement? If the difference is minor, I think the organization would make more sense without that distinction. if the difference is not minor, we should say what it is? or how it is operationally different for us?

> COMMITMENT: A value we hold and abide by.
> AGREEMENT: A specific action that we will perform as stated. 

For me, a commitment is essentially a standard by which I hold *myself* by.  Nobody else really knows the degree to which I am doing it. 

An agreement is more concrete and specific. Agreements are by their very nature measurable, so, one should be able to quantify the extent to which one is honoring an agreement (eg, I missed an hour with @j1c and @ebridge2 this week, so I made it up by scheduling another hour with them later).  

Does that make sense?

jovo commented:
@bdpedigo we could probably avoid 'family' or 'team' by just saying 'in NeuroData'?

bdpedigo commented:
> the actions associated with this are:
 recess
 vacations
 etc.

I would be happy to have a commitment to rest/recharching however works for us. And mentioning recess is also good. the rest of that section to me feels like interesting information but i dont see why it belongs on the agreements page, but happy to discuss with the lab further. 

> For me, a commitment is essentially a standard by which I hold myself by. Nobody else really knows the degree to which I am doing it.

> An agreement is more concrete and specific. Agreements are by their very nature measurable, so, one should be able to quantify the extent to which one is honoring an agreement 

sure, that makes sense. i see that you added something to differentiate the two, so consider me satisfied on this point 

> @bdpedigo we could probably avoid 'family' or 'team' by just saying 'in NeuroData'?

I have no issue with "team," i think that is appropriate. Happy to update everything to team, or just 'in NeuroData' as you suggest.

tathey1 commented:
I support removing the word "family," and I like the clarification between commitments (which here often seems to involve 1 person) and agreements (which often pertains to >1 people). 

I also prefer to omit the finer level description of what play. I think it's worth expressing that we value play, and giving recess as an example (after al, it caught the eye of @m-powell when he was considering joining), however I feel that much more than that will dilute the important ideas possibly confuse readers.

Should I make edits to address those points?

jovo commented:
> Should I make edits to address those points?

Yes please! This is *our* document, I *want* everyone to touch it to make it our own, not something that I just put forth.

jovo committed:
made some updates, but now think i may have created a merge conflict

jovo committed:
updating further to remove the delineation between commitments and agreements

jovo committed:
minor updates

jovo committed:
coming along

jovo committed:
.

jovo committed:
.

jovo committed:
.

jovo committed:
.

jovo committed:
.

jovo committed:
Update agreements.md

jovo committed:
.

jovo committed:
kinda done

jovo commented:
@bdpedigo i integrated what i thought was desirable from the code of conduct into the rest of the document, and then deleted the rest.  i currently believe the 'code of conduct' page is now superfluous.  happy to be overruled on that though.

jovo committed:
updating agreements

jovo committed:
formatting

jovo committed:
adding meeting structure

jovo committed:
replaced enumeration with bulletization

jovo committed:
fixed a bunch of typos

jovo committed:
removed some redundancy

jovo committed:
typos

tathey1 committed:
2 removals

- make external communication more concise
- remove responsibilities (they are now in the Other -> Lab Responsibilities page)

jovo committed:
Merge pull request #602 from neurodata/tathey1-patch-1

2 removals

m-powell committed:
basic proofreading edits

jshinm committed:
transcribe meeting format from jovo's slide

jshinm commented:
I've just modified the meeting section. Jovo's asked me to transcribe what's on his slide. I originally had this on a separate PR, but @tathey1 @PSSF23 nicely suggested that I move it here. Tommy wanted it shorter, and I'm all for it too, but before that, I wanted to know if everyone wants to keep this format or revert back to @m-powell's bullet point format, which I temporarily commented out. Please let me know :)

jshinm committed:
Merge branch 'deploy' into jovo-mid-updating-agreements

jovo committed:
added conservation of reviews.

jshinm committed:
Merge branch 'deploy' into jovo-mid-updating-agreements

jshinm reviewed (commented):
Please give your 👍 here if you can commit to our new agreement! If not, please propose further changes here 🙂

jshinm committed:
discontinue git protocol

jshinm committed:
update to use https protocol

jshinm committed:
clone over https

m-powell committed:
fixed a handful of typos

bdpedigo committed:
moved bit about meeting to guidance

bdpedigo committed:
Fix typo

bdpedigo committed:
Add bit about vacation from my convo with Jovo

bdpedigo commented:
Made a few small changes and gave my approval. 

One lingering comment is that  "@jovo paying our salaries" is under the section on commitments which "we expect to do 90% of the time." I feel that we all expect to get paid >90% of the time. So I just wonder if we should clarify that point on salary in particular?

jovo committed:
made salary a responsibility, not a commitment 

thanks for the suggestion @bdpedigo

jovo commented:
> One lingering comment is that "@jovo paying our salaries" is under the section on commitments which "we expect to do 90% of the time." I feel that we all expect to get paid >90% of the time. So I just wonder if we should clarify that point on salary in particular?

I fixed that in the above revision.

m-powell committed:
fixed a typo

m-powell committed:
another typo

jshinm reviewed (approved):
PR merging as per [discussion](https://neurodata.slack.com/archives/G72PSUVEX/p1648993500656729?thread_ts=1648780857.714869&cid=G72PSUVEX)
